opentelemetry-instrumentation/src/opentelemetry/instrumentation/utils.py

```diff
@@ -37,6 +37,10 @@
 
 propagator = TraceContextTextMapPropagator()
 
+_SUPPRESS_INSTRUMENTATION_KEY_PLAIN = (
```

Comment:
I wonder if this can be cleaned up now? https://github.com/open-telemetry/opentelemetry-python-contrib/blob/7249cc07301309748de26a4b1b941fad7f8d1551/opentelemetry-instrumentation/src/opentelemetry/instrumentation/utils.py

Reply:
I'm not sure if I got it. do you mean remove L40?

Reply:
Sorry I put the wrong link! I meant here https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-api/src/opentelemetry/context/__init__.py#L159-L165

Reply:
I found this issue https://github.com/open-telemetry/opentelemetry-python/issues/2184, which explains the motivation behind that. I think we don't have a clear decision on where this should be moved,  but I'm ok with keeping things as they are

Reply:
I think we still want to support the actual key object in case others have already taken a dependency